doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -696,26 +789,39 @@ this is just an example, not a statement of "best practices"):
 
     # Make all numpy available via shorter 'np' prefix
     import numpy as np
-    # Make all matlib functions accessible at the top level via M.func()
-    import numpy.matlib as M
-    # Make some matlib functions accessible directly at the top level via, e.g. rand(3,3)
-    from numpy.matlib import rand,zeros,ones,empty,eye
+    # 
+    # Make the SciPy linear algebra functions available as linalg.func()
+    # e.g. linalg.lu, linalg.eig (for general l*B@u==A@u solution)
+    from scipy import linalg
+    #
     # Define a Hermitian function
     def hermitian(A, **kwargs):
-        return np.transpose(A,**kwargs).conj()
-    # Make some shortcuts for transpose,hermitian:
-    #    np.transpose(A) --> T(A)
+        return np.conj(A,**kwargs).T
+    # Make a shortcut for hermitian:
     #    hermitian(A) --> H(A)
-    T = np.transpose
     H = hermitian
 
+To use the deprecated `matrix` and other `matlib` functions:
+
+::
+    
+    # Make all matlib functions accessible at the top level via M.func()
+    import numpy.matlib as M
+    # Make some matlib functions accessible directly at the top level via, e.g. rand(3,3)
+    from numpy.matlib import matrix,rand,zeros,ones,empty,eye
+
 Links
 =====
 
 See http://mathesaurus.sf.net/ for another MATLAB/NumPy
```

Comment:
This site is rather out of date regarding both python and numpy versions. Perhaps we should indicate that it is outdated.

Reply:
That's true, we can specify that its out of date. 

As an alternative link, there's a RealPython article on [MATLAB vs Python](https://realpython.com/matlab-vs-python/), maybe we could add this in the references.

Reply:
I'd be fine for now with just 
```
Another somewhat outdated MATLAB/NumPy cross-reference can be found at http://mathesaurus.sf.net/
```

Reply:
`Another somewhat outdated MATLAB/NumPy cross-reference can be found at http://mathesaurus.sf.net/`

updated reference in commit: de298885a4725889512e8172534d134b630c4da6